.github/workflows/fonts.yml

```diff
@@ -0,0 +1,111 @@
+name: Build fonts
```

Comment:
This file is based on https://github.com/KaTeX/katex-fonts/blob/master/.circleci/config.yml.